BetterUI/Main.cs

```diff
@@ -98,146 +100,152 @@ public Main()
         }
         public void Awake()
         {
+            string sectionName;
+
+            // Don't be tempted to rename the sections. That resets all its config values for every user
+            // The same goes for moving items between sections
+            // That's also why there is no section number counter
+
+
             // Player HUD
-            enablePlayerHudEditing = Config.Bind("1 - Player HUD", nameof(enablePlayerHudEditing), true, "Enable the ability to edit the Player HUD by pressing a hotkey. (REQUIRES RESTART)");
+            sectionName = "1 - Player HUD";
 
-            togglePlayerHudEditModeKey = Config.Bind("1 - Player HUD", nameof(togglePlayerHudEditModeKey), KeyCode.F7, "Key used to toggle Player HUD editing mode.");
+            togglePlayerHudEditModeKey = Config.Bind(sectionName, nameof(togglePlayerHudEditModeKey), KeyCode.F7,
+                "Key used to toggle Player HUD editing mode. Accepted values: https://docs.unity3d.com/ScriptReference/KeyCode.html");
 
-            modKeyPrimary = Config.Bind("1 - Player HUD", nameof( modKeyPrimary), KeyCode.Mouse0,
-              "Button needed to hold down to change HUD position. Accepted values: https://docs.unity3d.com/ScriptReference/KeyCode.html");
+            modKeyPrimary = Config.Bind(sectionName, nameof(modKeyPrimary), KeyCode.Mouse0,
+                "Key needed to hold down to change HUD position. Accepted values: https://docs.unity3d.com/ScriptReference/KeyCode.html");
 
-            modKeySecondary = Config.Bind("1 - Player HUD", nameof(modKeySecondary), KeyCode.LeftControl,
-              "Button needed to hold down to change element dimensions. Accepted Values: https://docs.unity3d.com/ScriptReference/KeyCode.html");
+            modKeySecondary = Config.Bind(sectionName, nameof(modKeySecondary), KeyCode.LeftControl,
+                "Key needed to hold down to change element dimensions. Accepted Values: https://docs.unity3d.com/ScriptReference/KeyCode.html");
 
-            useCustomHealthBar = Config.Bind("1 - Player HUD", nameof(useCustomHealthBar), false, "Resizable, rotatable HP bar. This bar will always be the same size and will not scale when you eat. (REQUIRES RESTART)");
-            
-            useCustomStaminaBar = Config.Bind("1 - Player HUD", nameof(useCustomStaminaBar), false, "Resizable, rotatable Stamina bar. This bar will always be visible and will not scale when you eat. (REQUIRES RESTART)");
+            // Player HUD RESTART
+            sectionName = "1 - Player HUD (Requires Logout)";
+
+            enablePlayerHudEditing = Config.Bind(sectionName, nameof(enablePlayerHudEditing), true, "Enable the ability to edit the Player HUD by pressing a hotkey.");
```

Comment:
Isn't having the (REQUIRES RESTART) in the setting description more visible than having it once in the section name? Not that people read it....

Reply:
the setting description is a text that appears when you hover on the config option while the mouse isn't controlling the player (like in the main menu or the pause menu or inside a container). it does not appear when hovering on the toggle itself, only the label. when you change a setting ingame without 'freeing' the mouse, then you don't get to see the description at all. for those reasons I think the description is the worst place to put this information. but of course... not that people read it. it's just an idea. in my personal opinion, the section label is a bit harder to miss